app/components/modals/tax-info-modal.js

```diff
@@ -49,56 +49,6 @@ export default ModalBase.extend(FormMixin, {
               prompt : this.l10n.t('Please give us your tax ID')
             }
           ]
-        },
-        taxInvoiceCompany: {
```

Comment:
@kushthedude I don't think that the issue specified anywhere to remove the validations.

Reply:
@uds5501 The validations which I removed were of optional fields of which tick box was recently removed, I dont think this fields should be mandatory unless and until event organiser is a firm.
@shreyanshdwivedi @mrsaicharan1 Your views ?